openmetadata-ui/src/main/resources/ui/src/components/BotDetails/BotDetails.component.tsx

```diff
@@ -248,70 +268,276 @@ const BotDetails: FC<BotsDetailProp> = ({
     );
   };
 
-  const getDescriptionComponent = () => {
+  const DescriptionComponent = ({ botData }: DescriptionComponentProps) => {
     return (
-      <div className="tw--ml-5">
-        <Description
-          description={botData.description || ''}
-          entityName={getEntityName(botData)}
-          hasEditAccess={descriptionPermission || editAllPermission}
-          isEdit={isDescriptionEdit}
-          onCancel={() => setIsDescriptionEdit(false)}
-          onDescriptionEdit={() => setIsDescriptionEdit(true)}
-          onDescriptionUpdate={handleDescriptionChange}
-        />
-      </div>
+      <Description
+        description={botData.description || ''}
+        entityName={getEntityName(botData)}
+        hasEditAccess={descriptionPermission || editAllPermission}
+        isEdit={isDescriptionEdit}
+        onCancel={() => setIsDescriptionEdit(false)}
+        onDescriptionEdit={() => setIsDescriptionEdit(true)}
+        onDescriptionUpdate={handleDescriptionChange}
+      />
```

Comment:
Since we aren't manipulating anything here use <Description /> directly to the render.
remove `DescriptionComponent` we don't need it